git commit doesn't work because tooling not built?

```
$ npm run test_patterns

> flowcrypt-browser 8.4.3 test_patterns
> node build/test/test/source/patterns.js

node:internal/modules/cjs/loader:936
  throw err;
  ^

Error: Cannot find module '/home/tom/git/flowcrypt-browser/build/test/test/source/patterns.js'
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12)
    at node:internal/main/run_main_module:17:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}
```

is there some setup I'm supposed to do? Could it happen automatically?

```
$ git commit -a -m "no issue - removed unused value"
✔ Preparing lint-staged...
❯ Running tasks for staged files...
  ❯ package.json — 1 file
    ❯ *.ts — 1 file
      ✔ npm run test_eslint
      ✖ npm run test_patterns [FAILED]
      ◼ prettier --write
    ↓ *.{css,htm} — no files [SKIPPED]
↓ Skipped because of errors from tasks. [SKIPPED]
✔ Reverting to original state because of errors...
✔ Cleaning up temporary files...

✖ npm run test_patterns:
node:internal/modules/cjs/loader:936
  throw err;
  ^

Error: Cannot find module '/home/luke/git/flowcrypt-browser/build/test/test/source/patterns.js'
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12)
    at node:internal/main/run_main_module:17:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}

> flowcrypt-browser@8.4.3 test_patterns
> node build/test/test/source/patterns.js "/home/luke/git/flowcrypt-browser/extension/js/common/api/email-provider/gmail/google-auth.ts"

husky - pre-commit hook exited with code 1 (error)
```

--- a/package.json
+++ b/package.json
@@ -64,6 +64,7 @@
     "build-assets-only": "./scripts/build.sh --assets-only",
     "pretest": "npm run build --silent && cd conf && node ../build/tooling/tsc-compiler --project tsconfig.test.json && node ../build/tooling/resolve-modules --project tsconfig.test.json",
     "pretest-incremental": "npm run build-incremental --silent && cd conf && node ../build/tooling/tsc-compiler --project tsconfig.test.json && node ../build/tooling/resolve-modules --project tsconfig.test.json",
+    "postinstall": "npm run pretest",
     "test": "npm run test_buf && npm run test_eslint && npm run test_patterns && npm run test_async_stack && npm run test_ci_chrome_consumer -- -- --pool-size=1",
     "test_local_chrome_consumer_mock": "npm run pretest && npm run test_ci_chrome_consumer -- -- --pool-size=1 --debug",
     "test_local_chrome_enterprise_mock": "npm run pretest && npm run test_ci_chrome_enterprise -- -- --pool-size=1 --debug",

